contracts/5/SushiswapPool12Goblin.sol

```diff
@@ -0,0 +1,246 @@
+pragma solidity 0.5.16;
+import "openzeppelin-solidity-2.3.0/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity-2.3.0/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity-2.3.0/contracts/utils/ReentrancyGuard.sol";
+import "openzeppelin-solidity-2.3.0/contracts/token/ERC20/IERC20.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol";
+import "@uniswap/v2-core/contracts/libraries/Math.sol";
+import "./uniswap/IUniswapV2Router02.sol";
+import "./Strategy.sol";
+import "./SafeToken.sol";
+import "./Goblin.sol";
+import "./interfaces/IMasterChef.sol";
+
+contract SushiswapPool12Goblin is Ownable, ReentrancyGuard, Goblin {
+    /// @notice Libraries
+    using SafeToken for address;
+    using SafeMath for uint256;
+
+    /// @notice Events
+    event Reinvest(address indexed caller, uint256 reward, uint256 bounty);
+    event AddShare(uint256 indexed id, uint256 share);
+    event RemoveShare(uint256 indexed id, uint256 share);
+    event Liquidate(uint256 indexed id, uint256 wad);
+
+    /// @notice Immutable variables
+    IMasterChef public masterChef;
+    IUniswapV2Factory public factory;
+    IUniswapV2Router02 public router;
+    IUniswapV2Pair public lpToken;
+    address public weth;
+    address public fToken;
```

Comment:
Why do we have fToken? It must be sushi right?

Reply:
Change `fToken` to `sushi`